app/components/Article/FeedbackForm/feedbackForm.css

```diff
@@ -0,0 +1,88 @@
+.feedback-header {
+  color: var(--colors-black);
+}
+.feedback-container {
+  border-radius: 6px;
+  background-color: #fff;
+  box-shadow: 0px 16px 40px rgba(175, 183, 194, 0.2);
+  border: 1px solid #dfe3e9;
+  box-sizing: border-box;
+  width: 384px;
+  height: 542px;
+  padding: var(--spacing-32);
+}
+.select-option {
+  border-radius: 6px;
+  background-color: #fff;
+  box-shadow: 0px 16px 40px rgba(175, 183, 194, 0.2);
+  border: 1px solid #dfe3e9;
```

Comment:
here too

As a general rule, too, if we're repeating code a lot we should just create a one-off class for newroot.css like

.border {
 border: 1px solid #dfe3e9;
}